android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -67,6 +73,13 @@
     private SearchView searchView;
     private Menu menu;
 
+    private Dialog upcomingDialogBox;
+    private ImageView trackImageIcon;
+    private TextView upcomingSessionText;
+    private TextView upcomingSessionTitle;
```

Comment:
Do we need all these global variables?

Reply:
Yes, these are used to describe the title and message. I think it should be there.